src/Sylius/Bundle/TranslationBundle/DependencyInjection/Compiler/TranslationListenerPass.php

```diff
@@ -24,19 +25,38 @@ class TranslationListenerPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        if (!$container->hasDefinition('sylius.translatable.listener')) {
-            return;
-        }
+        $driver = $container->getParameter('sylius_translation.driver');
```

Comment:
@stloyd I made a change to replace the locale listener argument when the driver is not orm on the TranslationListenerPass class. In my first intent I deleted the definition in the service.xml file and added a definition for the choosen driver in the TranslationListenerPass but I don't know why the loadClassMetadata and postLoad events are never invoked.